repos/cssparser.proj

```diff
@@ -0,0 +1,41 @@
+<Project>
+
+  <PropertyGroup>
+    <SourceDirectory>cssparser</SourceDirectory>
+  </PropertyGroup>
```

Comment:
This shouldn't be necessary if the source directory matches the project name like here.

Reply:
Thanks! Fixed now.